workflows/reference/gatk-reference.wdl

```diff
@@ -43,51 +48,63 @@ workflow gatk_reference {
         String dbSNP_vcf_url
         #@ except: SnakeCase
         String dbSNP_vcf_name
+        Int known_vcf_disk_size_gb
+        Int reference_fa_disk_size_gb
+        #@ except: SnakeCase
+        Int dbSNP_vcf_disk_size_gb
         #@ except: SnakeCase
         String? dbSNP_vcf_index_url
         #@ except: SnakeCase
         String? dbSNP_vcf_index_name
         String? interval_list_url
         String? interval_list_name
+        #@ except: SnakeCase
+        Int? dbSNP_vcf_index_disk_size_gb
+        Int? interval_list_disk_size_gb
     }
 
     call util.download as fasta_download { input:
         url = reference_fa_url,
         outfile_name = reference_fa_name,
-        md5sum = reference_fa_md5
+        md5sum = reference_fa_md5,
+        disk_size_gb = reference_fa_disk_size_gb,
```

Comment:
I was going to object to the workflow-level parameters, but since `disk_size_gb` is a required parameter without a default in `util.download`, I don't think we have much choice.